### PR TITLE
Tackle

### PR DESCRIPTION
**Pull request by Montana-Pfeifer**
added
#team_tackle_total
#most_tackles
#fewest_tackles

### CODE REVIEW

File: spec/stat_tracker_spec.rb [hunk 1]
```diff
@@ -168,4 +168,110 @@
...
+        "FC Dallas"=>0.32,
+        "Houston Dynamo"=>0.21,
+      }
+      expect(@stat_tracker1.team_shot_goal_ratios('20122013')).to include(expect)
+    end
+  end
+
+  describe '#most_accurate_team' do
+    it 'shows the team with the highest goal average' do
+      @stat_tracker1.most_accurate_team
```
Comment (wally-yawn) [inline spec/stat_tracker_spec.rb]: you shouldn't need this line, the next line should be handling everything it needs

File: spec/stat_tracker_spec.rb [hunk 2]
```diff
@@ -168,4 +168,110 @@
...
+      expect(@stat_tracker1.most_accurate_team).to eq("FC Dallas")
+    end
+
+    it 'shows most accurate team in the season' do
+      @stat_tracker1.most_accurate_team('20122013')
```
Comment (wally-yawn) [inline spec/stat_tracker_spec.rb]: you shouldn't need this line, the next line should be handling everything it needs

File: spec/stat_tracker_spec.rb [hunk 3]
```diff
@@ -168,4 +168,110 @@
...
+      expect(@stat_tracker1.most_accurate_team('20122013')).to eq("FC Dallas")
+    end
+  end
+
+  describe '#least_accurate_team' do
+    it 'shows the team with the lowest goal average' do
+      @stat_tracker1.least_accurate_team
```
Comment (wally-yawn) [inline spec/stat_tracker_spec.rb]: you shouldn't need this line, the next line should be handling everything it needs

File: spec/stat_tracker_spec.rb [hunk 4]
```diff
@@ -168,4 +168,110 @@
...
+      expect(@stat_tracker1.least_accurate_team).to eq("Atlanta United")
+    end
+  end
+
+  describe '#least_accurate_team by season' do
+    it 'shows least accurate team in the season' do
+      @stat_tracker1.least_accurate_team('20122013')
```
Comment (wally-yawn) [inline spec/stat_tracker_spec.rb]: you shouldn't need this line, the next line should be handling everything it needs

File: spec/stat_tracker_spec.rb [hunk 5]
```diff
@@ -168,4 +168,110 @@
...
+      expect(@stat_tracker1.least_accurate_team('20122013')).to eq("Atlanta United")
+    end
+  end
+
+  describe '#team_tackle_total' do
+    it 'makes a list of teams with tackle scores' do
+      @stat_tracker1.team_tackle_total
```
Comment (wally-yawn) [inline spec/stat_tracker_spec.rb]: you shouldn't need this line, the next line should be handling everything it needs

File: spec/stat_tracker_spec.rb [hunk 6]
```diff
@@ -168,4 +168,110 @@
...
+        "FC Cincinnati"=>0,
+        "DC United"=>0,
+        "FC Dallas"=>271,
+        "Houston Dynamo"=>179
+    }
+      expect(@stat_tracker1.team_tackle_total).to include(expect)
+    end
+
+    it 'makes a list of team tackles by season' do
+      @stat_tracker1.team_tackle_total('20122013')
```
Comment (wally-yawn) [inline spec/stat_tracker_spec.rb]: you shouldn't need this line, the next line should be handling everything it needs

File: spec/stat_tracker_spec.rb [hunk 7]
```diff
@@ -168,4 +168,110 @@
...
+        "DC United"=>0,
+        "FC Dallas"=>271,
+      }
+      expect(@stat_tracker1.team_tackle_total('20122013')).to include(expect)
+    end
+  end
+
+  describe '#most_tackles' do
+    it 'shows the team with the most tackles' do
+      @stat_tracker1.most_tackles
```
Comment (wally-yawn) [inline spec/stat_tracker_spec.rb]: you shouldn't need this line, the next line should be handling everything it needs

File: spec/stat_tracker_spec.rb [hunk 8]
```diff
@@ -168,4 +168,110 @@
...
+      expect(@stat_tracker1.most_tackles).to eq("FC Dallas")
+    end
+
+    it 'shows the team with the most tackles of the season' do
+      @stat_tracker1.most_tackles('20122013')
```
Comment (wally-yawn) [inline spec/stat_tracker_spec.rb]: you shouldn't need this line, the next line should be handling everything it needs

File: spec/stat_tracker_spec.rb [hunk 9]
```diff
@@ -168,4 +168,110 @@
...
+      expect(@stat_tracker1.most_tackles('20122013')).to eq("FC Dallas")
+    end
+  end
+
+  describe '#fewest_tackles' do
+    it 'shows the team with the fewest tackles' do
+      @stat_tracker1.fewest_tackles
```
Comment (wally-yawn) [inline spec/stat_tracker_spec.rb]: you shouldn't need this line, the next line should be handling everything it needs

File: spec/stat_tracker_spec.rb [hunk 10]
```diff
@@ -168,4 +168,110 @@
...
+      expect(@stat_tracker1.fewest_tackles).to eq("Atlanta United")
+    end
+
+    it 'shows the team with the fewest tackles of the season' do
+      @stat_tracker1.fewest_tackles('20122013')
```
Comment (wally-yawn) [inline spec/stat_tracker_spec.rb]: you shouldn't need this line, the next line should be handling everything it needs

**Review by wally-yawn — CHANGES_REQUESTED** [review]
I think there's an extraneous method call in your tests, I've added individual comments. They all look good though!

File: spec/stat_tracker_spec.rb (hunk 1 above)
Comment (Crosswolfv1) [reply spec/stat_tracker_spec.rb]: I agree as it isn't something they want us to test for in the interaction pattern they provided

File: spec/stat_tracker_spec.rb (hunk 3 above)
Comment (Crosswolfv1) [reply spec/stat_tracker_spec.rb]: I agree as it isn't something they want us to test for in the interaction pattern they provided

**Review by Crosswolfv1 — CHANGES_REQUESTED** [review]

**Review by Crosswolfv1 — APPROVED** [review]
Tests have been updated

**Review by wally-yawn — APPROVED** [review]

**Pull request merged**